### PR TITLE
Update Link component

### PR DESCRIPTION
**Pull request by danad02**
Linked to #2

### CODE REVIEW

**Comment by danad02** [comment]
verify

File: packages/ui/src/Link.tsx [hunk 1]
```diff
@@ -40,8 +41,8 @@ type LinkProps = IconProps & {
  * - Link can be used as a stand-alone component with right chevron icon.
  */
 export const Link: FC<LinkProps> = ({
-  size = 'regular',
-  Icon,
+  size = 'normal',
+  icon,
```
Comment (jankoritak) [inline packages/ui/src/Link.tsx:45]: Shouldn't we mirror Figma, as for naming? I would personally revert to the previous naming.

**Review by jankoritak — APPROVED** [review]
LGTM, leaving a comment regarding naming changes.

File: packages/ui/src/Link.tsx (hunk 1 above)
Comment (danad02) [reply packages/ui/src/Link.tsx:45]: I used `normal` and `small` because we have it everywhere - icon sizes, typography, ...

**Pull request merged**